JSQMessagesViewController/Views/JSQMessagesComposerTextView.m

```diff
@@ -156,6 +156,17 @@ - (void)setTextAlignment:(NSTextAlignment)textAlignment
     [self setNeedsDisplay];
 }
 
+- (void)paste:(nullable id)sender
```

Comment:
let's remove `nullable` for now.

I'm going to take a big pass soon to add nullability annotations.